app/views/dormant_user_mailer/email_dormant_user.text.erb

```diff
@@ -0,0 +1,22 @@
+Dear <%= @user.display_name %>,
+
+Our citizen science projects need you!
+
+Whether it is helping to classify galaxies from satelite images or lending a hand to transcribe anti-slavery manuscripts, there are many exciting research projects to get involved with in the Zooniverse.
+
+Check out all our projects here: (https://zooniverse.org/#/projects)
+
+And don't forget to log in so you can keep track of your contributions and get involved on the project discussion boards (https://zooniverse.org/#/talk) and keep track of your contributions and exciting discoveries.
+
+See you out there!
+
+The Zooniverse Team
+
+UNSUBSCRIBE INSTRUCTIONS:
+You are receiving this email because you have recently registered an account with the Zooniverse.
+<%- if @user %>
+To unsubscribe to these messages instantly visit <%= unsubscribe_url(token: @user.unsubscribe_token) %>
+Please be aware that this action will unsubscribe you from ALL Zooniverse emails; if you wish to manage individual project subscriptions, please visit your settings page link below.
+Alternatively you can <a href="https://zooniverse.org/#/unsubscribe">unsubscribe here</a>.
```

Comment:
We can't use the HTML tags in a text email, e.g. `<a>` etc. Construct the text link instead. If this was copied we can fix the other templates in another PR.